Build/build.cmd

```diff
@@ -7,51 +7,62 @@ IF EXIST "..\ext" (
     rmdir ..\ext /s /q    
 )
 
-REM Checkout OpenDDS to the version 3.13.2
-TITLE Checkout OpenDDS to the version 3.13.2
+REM Unset enviroment Variables
+TITLE Unset enviroment variables
+set MPC_ROOT=
+set ACE_ROOT=
+set TAO_ROOT=
+set DDS_ROOT=
+
+REM Set VSINSTALLDIR
+TITLE Set VSINSTALLDIR
+set VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\
```

Comment:
`VSINSTALLDIR` should point to the `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\` folder and not `VC` subfolder.

We could put a comment to inform that the variable must be adjusted depending of your VS version installation path.